diff --git a/node_modules/typecov/lib/index.js b/node_modules/typecov/lib/index.js
index 2010f7c..497ea86 100644
--- a/node_modules/typecov/lib/index.js
+++ b/node_modules/typecov/lib/index.js
@@ -30,7 +30,7 @@ function getReport(headTypeCoverageArtifact, baseTypeCoverageArtifact, options)
         : 0;
     const coverageDiff = headTypeCoverage - baseTypeCoverage;
     const newUntypedSymbols = findNew(headTypeCoverageArtifact.allUntypedSymbols, baseTypeCoverageArtifact ? baseTypeCoverageArtifact.allUntypedSymbols : []);
-    const status = options.atLeast === undefined || headTypeCoverage > options.atLeast ? "success" : "failure";
+    const status = options.atLeast === undefined || headTypeCoverage >= options.atLeast ? "success" : "failure";
     const shortDescription = formatters_1.formatShortDescription({
         coverageDiffPerc: coverageDiff,
         headTypeCoveragePerc: headTypeCoverage,
